include/linear_feedback_controller_msgs/eigen_conversions.hpp

```diff
@@ -16,6 +16,7 @@ struct JointState {
   ::Eigen::VectorXd position;
   ::Eigen::VectorXd velocity;
   ::Eigen::VectorXd effort;
+  rclcpp::Time stamp;
```

Comment:
I would use std::chrono as much as possible to be honest...

Reply:
I thought about it. However, since we use it only for ROS related things, I don't think it makes sense to convert it back and forth.

Reply:
I would prefer to add the stamp into the "Sensor" and "Control" structs.
I makes more sense to me as all sensors are synchronized under the top header message.